tools/nvme.sh

```diff
@@ -17,6 +17,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+export TIMEFORMAT='%3lR'
+
 usage() {
```

Comment:
Do we always want this? I could see this being an optional argument `-t` or something.

See rabbit-s.sh for an example - it uses SSHPASS if you provide `-p`